lib/assertions.js

```diff
@@ -231,8 +231,7 @@ const oidc = {
       nonce,
       accessToken = crypto.randomBytes(15).toString('hex'),
     ) => {
-      const nric = oidc.singPass[uuid]
-      const sub = `s=${nric},u=${uuid}`
+      const sub = `u=${uuid}`
```

Comment:
Hang on, this impacts singpass. Why remove the `s=`?

Reply:
will remove the s= from the sgid script instead